login_V1

```diff
@@ -40,47 +35,100 @@
 		
 		<div class="grid-container">
 			
-			<div class="nav" style="background-image: url('8.jpg'); background-size: 100vw 60vw;">
-				<ul>
-					<li><a href="index.html">ZEPHYRUS</a></li>
-				</ul>
-			</div>
+			<?php
+				require 'loading.php';
+			?>
 			
-			<div class="content" style="background-image: url('8.jpg'); background-size: 100vw 60vw;">
+			<div class="content" style="background-image: url('images/img_20.jpg'); background-size: 100vw 58vw;">
 			
+				
+				<div class="nav" style="z-index: 2; background-color:rgba(0, 0, 0, 0.4);">
+					<ul>
+						<li><a href="index.php" style="color: white;"><img src="images/icon_5.png" style="height: 1.95vw; width: 1.95vw; margin-right: 0.5vw; margin-bottom: -0.4vw;" draggable="false">ZEPHYRUS</a></li>
+					</ul>
+				</div>
+				
+				
 				<div class="scroll-area">
 					
-					<h1>Feel The Beat.</h1>
-					<h2>Find your Journey</h2>
+					<h1 style="font-size: 4.6vw;">Matrix Mainframe.</h1>
+					<h2 style="margin-left: 9.2vw; margin-top: -1vw; font-size: 2.5vw;">Graeme's Music Settings Block</h2>
 				
 				</div>
+				
+				<div class="table" style="background-color: transparent; color: white; width: 40vw; height: 15vw; margin: 25vw 0vw 10vw 8vw; overflow-y: scroll;">
+					
+					<div class="row" style="display: grid; grid-template-columns: 1fr 1fr; background-color: white; color: darkgreen; border-bottom: 0.1vw solid grey; transition: 0.15s ease-in-out;">
+
+							<div class="ID" style="grid-column: 1;"><p style="margin: 0.5vw 2vw; font-size: 1vw;">Username</p></div>
+
+							<div class="Title" style="grid-column: 2; border-right: 0.1vw solid grey;"><p style="margin: 0.5vw; font-weight: bold; font-size: 1vw;">Password</p></div>
+
+						</div>
+					
+					<?php
```

Comment:
The User Delete Page here contains a query that lists all of the users in the database.